common/src/main/java/io/netty5/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -379,7 +379,7 @@ protected int runAllTasks(int maxTasks) {
      */
     protected final long delayNanos(long currentTimeNanos) {
         assert inEventLoop();
-        currentTimeNanos -= START_TIME;
+        currentTimeNanos -= SystemTicker.START_TIME;
```

Comment:
Greetings!
I have a suggestion regarding the code. Would it be possible to use `ticker().initialNanoTime()` to improve its testability? Thanks for your time!

```suggestion
        currentTimeNanos -= ticker().initialNanoTime();
```

Reply:
Sure. That's a good point!